recipes/assertpy/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "assertpy" %}
+{% set version = "0.12" %}
+{% set sha256 = "563502ad2d89f5242543924d1ae62472c24daeeaf2ecfd78b004feb0559395b9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --record record.txt
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+
+test:
+  imports:
+    - assertpy
+
+about:
+  home: https://github.com/ActivisionGameScience/assertpy  
+  license: BSD 3-Clause
+  license_file: LICENSE
```

Comment:
Since there is no `MANIFEST.in`, the `LICENSE` file isn't included in the source distribution on pypi.io. You'll need to do the following:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually